### PR TITLE
Micro aquisition deck for PPOC

### PR DESCRIPTION
**Pull request by RachelMuston**
This is the first draft of the deck.

https://sara-sabr.github.io/util-presentation/presentation.html?url=https://raw.githubusercontent.com/sara-sabr/ITStrategy/Micro-aquisition_PPOC/presentations/en/Micro-acquisition_PPOC.md

### CODE REVIEW

**Review by gcharest — APPROVED** [review]
I found the deck very clear and, with the speaker notes, covering all the topics I could think of. I know there were a few more added in the chat but other than that, I'd approve the presentation :+1:

**Comment by ElminaIusifova** [comment]
I have approved the PR  but a small suggestion: why not show the procedure of MA starting from the opportunity posted on the stand-alone website and ending with payment to a winning supplier for work done to make the content more digestible?

**Review by ElminaIusifova — APPROVED** [review]

File: presentations/en/Micro-acquisition_PPOC.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+
+## Stakeholders
+
+- **Executive Sponsors:** Mathieu Leprohon (IITB) and Robert MacGillivary (CFOB)
+- **IITB Lead:** Martin Mondor
+- **Proposal Contact:** Rachel Muston
+- **Client Lead:** Matthew Diedrick, Hugo Desgent-Barbe
+- **BRM Contact:** Amos Lee
+- **EA Contact:** Erin Rowlinson
+- **Others:** Suppliers, ESDC clients, Office of Small and Medium Enterprise (PSPC), PSPC Legal (*)
```
Comment (gabrielcossette) [inline presentations/en/Micro-acquisition_PPOC.md:22]: Is the "(*)" for a future footnote?

File: presentations/en/Micro-acquisition_PPOC.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+   <th>Total Salary</th>
+   <th>Non-Salary</th>
+   <th>Total</th>
+ </tr>
+ <tr>
+   <td>2020-21</td>
+   <td>$81K</td>
+   <td>$15K</td>
+   <td>$96K</td>
+   <td>$0.015</td>
```
Comment (gabrielcossette) [inline presentations/en/Micro-acquisition_PPOC.md:125]: We need to disclose a cost of 1,5 cents?

File: presentations/en/Micro-acquisition_PPOC.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+- "development complete" includes not just the development of the technical solution but also the development of the processes, the generic terms and conditions, the user documentation, the audit and monitoring process, communication pieces
+
+---
+
+## Solution Architecture and Design
+
+All existing products and/or Approved Tech Bricks
+
+- Stand-alone website to post opportunities
+- Hosting via GitHub pages
```
Comment (gabrielcossette) [inline presentations/en/Micro-acquisition_PPOC.md:175]: Should we specify "Free hosting"?

**Review by gabrielcossette — APPROVED** [review]
Great work! Approved as well. Did minor tweaks and added link to blog post pt2 (published after this draft) :)

File: presentations/en/Micro-acquisition_PPOC.md (hunk 1 above)
Comment (RachelMuston) [reply presentations/en/Micro-acquisition_PPOC.md:22]: Good catch! Was going to add a footnote but after a convo with CFOB procurement today it looks very likely we will need to involve legal (could be ESDC or PSPC legal). I've adjusted the wording and removed the '(*)'

File: presentations/en/Micro-acquisition_PPOC.md (hunk 2 above)
Comment (RachelMuston) [reply presentations/en/Micro-acquisition_PPOC.md:125]: Another good catch! There is a 'K' missing from that line.  I've made that fix. So its $15 and not 1.5 cents. Still not really worthy of disclosing but it is in the costing worksheet and so figured the two things would need to match. :-)

File: presentations/en/Micro-acquisition_PPOC.md (hunk 3 above)
Comment (RachelMuston) [reply presentations/en/Micro-acquisition_PPOC.md:175]: Yes! I have added 'Free'. :)

**Pull request merged**